sdk/search/search-documents/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 11.3.0 (2021-09-??)
```

Comment:
The catch with going with 11.3.0 here is that we already have betas out for this version 
I think 11.3.0-beta.1 and 11.3.0-beta.2 :(

Reply:
Yea, unfortunately I noticed this as well. What do you and @sarangan12 think we should do in that case? Going to 11.4.0 feels weird, since it'll mean a gap, but going to 11.3.0 feels weird because of the existing betas...

Reply:
Oh! I also considered 11.2.1 but we historically have done a minor version bump when updating code-tracing and core-http

Reply:
We could go with 11.3.0 and then have @sarangan12 release a 11.4.0-beta.1 instead of 11.3.0-beta.3 like he was planning to do next week. The changelog should then call out that 11.3.0 will not be including any of the features from the 11.3.0-betas and for folks who are looking for those features, they should wait for 11.4.0-beta.1

Thoughts?

Reply:
I think that's reasonable, personally, and I can make the changes to the CHANGELOG shortly. As long as @sarangan12 is good with the plan I'm 👍

Reply:
> Let me get it clear:
> ....
> 2. Do the same fix for the next week's release also in 11.3.0-beta.3

The only thing that is not needed is ^ assuming you're releasing from main. Everything else is correct 👍 

As far as the version - I actually don't know the semver policies here but I can do whatever y'all think makes sense - I have no stake in this other than getting customers on the latest OpenTelemetry for GA'd packages

Reply:
11.2.1 is fine by me

Reply:
11.2.1 it is!